relation_engine_server/test/spec_release/sample_spec_release/spec/collections/ncbi/ncbi_taxon.yaml

```diff
@@ -5,6 +5,7 @@ delta: true
 indexes:
   - type: fulltext
     fields: [scientific_name]
+    minLength: 1
```

Comment:
All the fulltext indexes can get a new param `minLength=1` in order to work well in searches